extension-requests/script.js

```diff
@@ -162,19 +162,50 @@ const intersectionObserver = new IntersectionObserver(async (entries) => {
   }
 });
 function handleSuccess(element) {
-  element.classList.add('success-card');
-  setTimeout(() => element.classList.remove('success-card'), 1000);
+  element.classList.add('green-card');
+  setTimeout(() => element.classList.remove('green-card'), 1000);
 }
 function handleFailure(element) {
-  element.classList.add('failed-card');
-  setTimeout(() => element.classList.remove('failed-card'), 1000);
+  element.classList.add('red-card');
+  setTimeout(() => element.classList.remove('red-card'), 1000);
 }
-async function removeCard(element) {
-  element.classList.add('success-card');
+async function removeCard(element, style) {
```

Comment:
the `style` variable name can be improved I think, eg. `className` or `elementClass`